x-pack/plugin/watcher/src/test/java/org/elasticsearch/xpack/watcher/condition/ScriptConditionTests.java

```diff
@@ -206,7 +206,7 @@ public void testScriptConditionAccessCtx() throws Exception {
         assertThat(condition.execute(ctx).met(), is(true));
     }
 
-    public void testParamsCtxDeprecated() throws Exception {
+    public void testParamsCtxNull() throws Exception {
```

Comment:
I wonder if this test (and the one below) still make sense ? 

The test is based on the absence of something that used to be there. I would be +1 to simply remove the deprecation tests (instead of changing them to null tests)

Reply:
I think its fair to remove these 2 tests (this and the other one mentioned i forgot to rename).